facade/service.go

```diff
@@ -621,6 +621,12 @@ func (f *Facade) validateServiceDeployment(ctx datastore.Context, parentID strin
 			glog.Errorf("Could not create service %s: %s", sd.Name, err)
 			return nil, err
 		}
+		// Evaluate templated endpoints
+		if err = f.evaluateEndpointTemplates(ctx, svc); err != nil {
+			glog.Errorf("Could not evaluate endpoint templates for service %s with parent %s: %s", svc.Name, svc.ParentServiceID, err)
```

Comment:
Per the [logging guidelines](https://sites.google.com/a/zenoss.com/engineering/home/faq/logging-guidelines), we should not both log and return an error.